src/main/scala/viper/silver/parser/ParseAst.scala

```diff
@@ -231,7 +240,9 @@ trait PIdentifier extends PLeaf {
   override def display = name
 }
 
-case class PIdnDef(name: String)(val pos: (Position, Position)) extends PNode with PIdentifier
+case class PIdnDef(name: String)(val pos: (Position, Position)) extends PNode with PIdentifier {
+  override def reformat(implicit ctx: ReformatterContext): List[RNode] = rt(name)
```

Comment:
Move to `PIdentifier` and remove from other classes implementing the trait. Is it not possible to reuse the `PLeaf` and its `display` function to avoid `reformat` implementation for all classes implementing `PLeaf`?